### ISSUE TITLE
BulkRead with IsRowVersion() property throws System.InvalidOperationException: The given ColumnMapping does not match up with any column in the source or destination.

### ISSUE THREAD

**Issue by HenkKin** [borisdj/EFCore.BulkExtensions#533]
See my fiddle: https://dotnetfiddle.net/EWQNJT

When using an entity 

```     
public class Customer
{ 
     public byte[] RowVersion { get; set; }
}
```

With EF Core configuration

```
 modelBuilder.Entity<Customer>(entity =>
 {
     // This is not working
     entity.Property(p=>p.RowVersion).IsRowVersion();
	
     // This is working
     //entity.Property(p=>p.RowVersion); 
		
     // so the issue seems to be IsRowVersion()
  });
```

And using

```
// We have a list of customers with only name is provided and we want to get the Id column filled based on Name column
var customers = new List<Customer>{ new Customer { Name = "Customer_B" } };

context.BulkRead(customers, new BulkConfig{
     // PropertiesToExclude = new List<string>{ nameof(Customer.RowVersion) }, // This results in same error
     // PropertiesToInclude = new List<string>{ nameof(Customer.Id) }, // This results in same error
     UpdateByProperties = new List<string>{ nameof(Customer.Name) }
});
```

Results into exception:

```
Unhandled exception. System.InvalidOperationException: The given ColumnMapping does not match up with any column in the source or destination.
   at Microsoft.Data.SqlClient.SqlBulkCopy.AnalyzeTargetAndCreateUpdateBulkCommand(BulkCopySimpleResultSet internalResults) in /_/src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs:line 726
   at Microsoft.Data.SqlClient.SqlBulkCopy.WriteToServerInternalRestContinuedAsync(BulkCopySimpleResultSet internalResults, CancellationToken cts, TaskCompletionSource`1 source) in /_/src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs:line 2810
   at Microsoft.Data.SqlClient.SqlBulkCopy.WriteToServerInternalRestAsync(CancellationToken cts, TaskCompletionSource`1 source) in /_/src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs:line 3042
   at Microsoft.Data.SqlClient.SqlBulkCopy.WriteToServerInternalAsync(CancellationToken ctoken) in /_/src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs:line 3101
   at Microsoft.Data.SqlClient.SqlBulkCopy.WriteRowSourceToServerAsync(Int32 columnCount, CancellationToken ctoken) in /_/src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs:line 2037
   at Microsoft.Data.SqlClient.SqlBulkCopy.WriteToServer(DataTable table, DataRowState rowState) in /_/src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs:line 1778
   at Microsoft.Data.SqlClient.SqlBulkCopy.WriteToServer(DataTable table) in /_/src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs:line 1751
   at EFCore.BulkExtensions.SQLAdapters.SQLServer.SqlOperationsServerAdapter.InsertAsync[T](DbContext context, Type type, IList`1 entities, TableInfo tableInfo, Action`1 progress, CancellationToken cancellationToken, Boolean isAsync) in D:\ProjectsPriv\EFCore.BulkExtensions\EFCore.BulkExtensions\SQLAdapters\SQLServer\SqlServerAdapter.cs:line 63
   at EFCore.BulkExtensions.SQLAdapters.SQLServer.SqlOperationsServerAdapter.InsertAsync[T](DbContext context, Type type, IList`1 entities, TableInfo tableInfo, Action`1 progress, CancellationToken cancellationToken, Boolean isAsync) in D:\ProjectsPriv\EFCore.BulkExtensions\EFCore.BulkExtensions\SQLAdapters\SQLServer\SqlServerAdapter.cs:line 90
   at EFCore.BulkExtensions.SQLAdapters.SQLServer.SqlOperationsServerAdapter.InsertAsync[T](DbContext context, Type type, IList`1 entities, TableInfo tableInfo, Action`1 progress, CancellationToken cancellationToken, Boolean isAsync) in D:\ProjectsPriv\EFCore.BulkExtensions\EFCore.BulkExtensions\SQLAdapters\SQLServer\SqlServerAdapter.cs:line 101
   at EFCore.BulkExtensions.SQLAdapters.SQLServer.SqlOperationsServerAdapter.ReadAsync[T](DbContext context, Type type, IList`1 entities, TableInfo tableInfo, Action`1 progress, CancellationToken cancellationToken, Boolean isAsync) in D:\ProjectsPriv\EFCore.BulkExtensions\EFCore.BulkExtensions\SQLAdapters\SQLServer\SqlServerAdapter.cs:line 323
   at EFCore.BulkExtensions.SQLAdapters.SQLServer.SqlOperationsServerAdapter.ReadAsync[T](DbContext context, Type type, IList`1 entities, TableInfo tableInfo, Action`1 progress, CancellationToken cancellationToken, Boolean isAsync) in D:\ProjectsPriv\EFCore.BulkExtensions\EFCore.BulkExtensions\SQLAdapters\SQLServer\SqlServerAdapter.cs:line 359
   at EFCore.BulkExtensions.SQLAdapters.SQLServer.SqlOperationsServerAdapter.Read[T](DbContext context, Type type, IList`1 entities, TableInfo tableInfo, Action`1 progress) in D:\ProjectsPriv\EFCore.BulkExtensions\EFCore.BulkExtensions\SQLAdapters\SQLServer\SqlServerAdapter.cs:line 293
   at EFCore.BulkExtensions.SqlBulkOperation.Read[T](DbContext context, Type type, IList`1 entities, TableInfo tableInfo, Action`1 progress) in D:\ProjectsPriv\EFCore.BulkExtensions\EFCore.BulkExtensions\SqlBulkOperation.cs:line 55
   at EFCore.BulkExtensions.DbContextBulkTransaction.Execute[T](DbContext context, Type type, IList`1 entities, OperationType operationType, BulkConfig bulkConfig, Action`1 progress) in D:\ProjectsPriv\EFCore.BulkExtensions\EFCore.BulkExtensions\DbContextBulkTransaction.cs:line 35
   at EFCore.BulkExtensions.DbContextBulkExtensions.BulkRead[T](DbContext context, IList`1 entities, BulkConfig bulkConfig, Action`1 progress, Type type) in D:\ProjectsPriv\EFCore.BulkExtensions\EFCore.BulkExtensions\DbContextBulkExtensions.cs:line 138
   at Program.Main()
```

See my fiddle: https://dotnetfiddle.net/EWQNJT

**Comment by borisdj** [comment]
Fixed with v5.1.1

**Comment by HenkKin** [comment]
Thnx for quick solution
